Project.toml

```
@@ -23,7 +23,7 @@ msolve_jll = "6d01cc9a-e8f6-580e-8c54-544227e08205"
 AbstractAlgebra = "0.24.1"
 DocStringExtensions = "0.8"
 GAP = "0.7.7"
-Hecke = "0.11.1"
+Hecke = "0.12.1"
```

There is no 0.12.1 yet, though? Perhaps you meant this?
```suggestion
Hecke = "0.12.0"
```